x-pack/plugins/endgame/public/application/endpoints/endpoint_list_view_1.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { PureComponent } from 'react';
+
+import { EuiBasicTable } from '@elastic/eui';
+import { EndgameAppContext } from '../../common/app_context';
+import { Page } from '../../components/page';
+import { SearchBar } from '../../components/search_bar';
+
+const columns = [
+  {
+    field: '_source',
```

Comment:
@parkiino cleaned this up a bit in her PR with the field names.  The data was formatted a bit differently as well

Reply:
@kevinlog - was it merged? maybe I just need to pull `upstream` back into my repo.